layouts/learning-paths/baseof.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{ partial "head.html" . }}
+</head>
+
+<body class="flex flex-col min-h-screen bg-background-light text-base dark:bg-background-dark dark:text-white">
+  {{ partial "header.html" . }}
+  <main class="flex justify-center">
+    <div class="lg:w-[1200px] overflow-clip p-6 pt-0 w-lvw">
+      {{ block "main" . }}
+      {{ end }}
+    </div>
+  </main>
+  <footer class="mt-auto">{{ partialCached "footer.html" . }}</footer>
+  {{ with (.Store.Get "youtube") }}
```

Comment:
duplicated because this is a new `baseof`. Should either put this in a partial, or move it to the footer partial which is reused everywhere.